### ISSUE TITLE
`>` next to folder name

### ISSUE THREAD

**Issue by hsiktas** [aidenlx/alx-folder-note#34]
I noticed that the plugin now displays a `>` after each folder name.

![image](https://user-images.githubusercontent.com/4571159/142909956-5542827a-f61c-432a-8d61-bea2fa1ac4b1.png)

Is this intentional? What is its purpose?

**Comment by aidenlx** [comment]
It’s because the old underline style would conflict with the new folder icon feature, if you want the underlines, you can add the following css snippet:

**Comment by laurastephsmith** [comment]
This seems to have reverted to underline with the update to 0.15.0 - is that intentional?

**Comment by aidenlx** [comment]
yes, that is intentional, I brought them back because I've figure out how to avoid conflict with the folder icon for underline style. You can still use the `>` style with css snippet, btw: 

```css
.alx-folder-icons
  .nav-folder.alx-folder-with-note
  > .nav-folder-title
  > .nav-folder-title-content::after {
  content: ">";
  color: var(--text-accent);
  margin-left: 5px;
}

.alx-folder-icons
  .nav-folder.alx-folder-with-note[data-icon]
  > .nav-folder-title
  > .nav-folder-title-content::before {
  display: none;
}
```

**Comment by laurastephsmith** [comment]
Thanks for the CSS. I'm kinda getting used to the underline now, but I'll keep this in mind!
